server/utils/ssr.tsx

```diff
@@ -221,8 +235,7 @@ export const generateMetaComponents = (metaProps: MetaProps) => {
 			<meta key="pa1" property="article:published_time" content={String(publishedAt)} />,
 			<meta key="pa2" property="dc.date" content={dcPublishedAt} />,
 			<meta key="pa3" name="citation_publication_date" content={googleScholarPublishedAt} />,
-			<meta key="pub1" name="citation_publisher" content="PubPub" />,
-			<meta key="pub2" property="dc.publisher" content="PubPub" />,
+			<meta key="pub1" property="dc.publisher" content="PubPub" />,
```

Comment:
should i add meta data for conference as well?

Reply:
yes!

Also, sorry for adding a new thing, but I just realized the `dc.publisher` should also change to be whatever's in the Publisher field. That's properly where Zotero, etc. have been grabbing the publisher from.